layouts/default.vue

```diff
@@ -40,6 +40,13 @@ export default {
     currentPage() {
       return this.$router.path
     }
+  }, mounted() {
```

Comment:
apparently not required 
https://github.com/nuxt-community/modules/pull/49#event-1144717409